src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,290 @@
 
 * [Bug Report Tool](https://bugreport.java.com/)
 * [JDK Bug System (JBS)](https://bugs.openjdk.org/)
-:::
+  :::
+
+[JBS](https://bugs.openjdk.org/) is a public issue tracker used by many OpenJDK projects and is open for anyone to read and search. In order to get write access you need to be registered in the [OpenJDK Census](https://openjdk.org/census) by becoming, for instance, an [Author](https://openjdk.org/bylaws#author) in an OpenJDK [Project](https://openjdk.org/bylaws#project).
+
+## Filing an issue
+
+When a new failure is found, or an improvement identified, an issue should be filed to describe and track its resolution.
+Depending on your role in OpenJDK you can either use the [Bug Report Tool](https://bugreport.java.com/) or, if you
+are registered in the [OpenJDK Census](https://openjdk.org/census), you can report the issue directly in [JBS](https://bugs.
+openjdk.org/).
 
-[JBS](https://bugs.openjdk.org/) is a public issue tracker used by many OpenJDK projects. It's open for anyone to read and search. In order to get write access you need to be registered in the [OpenJDK Census](https://openjdk.org/census), for instance by becoming an [Author](https://openjdk.org/bylaws#author) in an OpenJDK [Project](https://openjdk.org/bylaws#project).
+When filing an Issue, try to make the report as complete as possible in order to make it easier to triage, investigate and resolve the issue.
 
-## Filing a bug
+#### Types of Issues
 
-When a new failure is found in the JDK a bug should be filed to describe and track the issue. Depending on your role in OpenJDK you can either use the [Bug Report Tool](https://bugreport.java.com/) or, if you are registered in the [OpenJDK Census](https://openjdk.org/census), report the bug directly in [JBS](https://bugs.openjdk.org/). Try to make the bug report as complete as possible to make it easier to triage and investigate the bug.
+The most common Issue types are:
+<br />
+<table class="dictionary" summary="JBS Label Dictionary">
+  <tr style="text-align:left;"><th>Type</th><th>Covers</th></tr>
+<tr>
+    <td class="dictionary">[Bug]{.jbs-field}</td>
+    <td class="dictionary">Used when reporting a problem: crashes; hangs; failures of functionality etc.</td>
+</tr>
+<tr>
+    <td class="dictionary"><i>Improvement</i></td>
```

Comment:
When you go to create a bug in JBS, the issue types available include "Bug", "Enhancement", "JEP", etc. not "Improvement" -- the guide should match JBS terminology.

Reply:
Agreed the table tries to do too much and so it is confusing as the issue types are in both columns. I've changed it to be simpler, moving the part about vulnerabilities to under the table.